docs/CreateEdit.md

```diff
@@ -1281,10 +1286,11 @@ export const UserCreate = (props) => (
             <TextInput label="Email" source="email" validate={validateEmail} />
             <TextInput label="Age" source="age" validate={validateAge}/>
             <TextInput label="Zip Code" source="zip" validate={validateZipCode}/>
-            <SelectInput label="Sex" source="sex" choices={[
+            <SelectInput label="Genre" source="genre" choices={[
```

Comment:
Typo `Genre` => `Gender`

Reply:
You don't even wait for merged docs anymore :joy: Thanks!

Reply:
Tried to be more inclusive and ended up being even less than before :scream: